internal/handlers/handlers.go

```diff
@@ -12,9 +13,18 @@ import (
 )
 
 func FirstHandler(w http.ResponseWriter, r *http.Request) {
-	data, err := os.ReadFile("../index.html")
+	wd, err := os.Getwd()
 	if err != nil {
-		http.Error(w, "Ошибка чтения файла", http.StatusInternalServerError)
+		http.Error(w, "Ошибка получения текущей дирректории", http.StatusInternalServerError)
+		return
+	}
+
+	fullPath := filepath.Join(wd, "..", "index.html")
```

Comment:
Ошибка тут, запускать нужно из корня проекта командой `go run cmd/main.go`
```suggestion
	fullPath := "index.html"
```